lib/modules/manager/pip-compile/extract.ts

```diff
@@ -137,8 +160,7 @@ export async function extractAllPackageFiles(
         });
       } else {
         logger.warn(
-          { packageFile },
-          'pip-compile: failed to find dependencies in source file',
+          `pip-compile: failed to find dependencies in source file ${packageFile}`,
```

Comment:
Please revert. We want to avoid string interpolation in WARN or ERROR messages so that there is a finite number of such messages and we can one day have them all authoritatively documented